web/src/app/alerts/CreateAlertDialog/StepContent/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import gql from 'graphql-tag'
+import { useQuery } from 'react-apollo'
+import _ from 'lodash-es'
+import Step0 from './Step0'
+import Step1 from './Step1'
+import Step2 from './Step2'
+import Step3 from './Step3'
+
+const query = gql`
+  query($input: ServiceSearchOptions) {
+    services(input: $input) {
+      nodes {
+        id
+        name
+        isFavorite
+      }
+    }
+  }
+`
+
+export default props => {
+  const { formFields, mutationStatus, onChange } = props
+
+  // TODO loading error handles
+  const { data } = useQuery(query, {
+    variables: {
+      input: {
+        search: formFields.searchQuery,
+        favoritesFirst: true,
+        omit: formFields.selectedServices,
+        favoritesOnly: formFields.searchQuery.length === 0,
```

Comment:
We can drop this since we already have `favoritesFirst`

Reply:
If there are no favorites it would default to showing the first services A-Z sorted. I'm kind of in favor of showing only favorites, and searching for the relevant content you want if not applicable

Reply:
Keep in mind not all users of GoAlert will have a large number of services. Also, search-selects currently use `favoritesFirst` as does the search page. Not all users will have favorites either, depending on how they use GoAlert.

Giving some data (A-Z) rather than no data also gives the user a "hint" at what to search for by seeing existing names before entering a query.

Lastly, fav.-only may be a little cleaner in some use-cases, but it would also be different behavior than the rest of the app as well as changing behavior here based on what's in the search field.